src/components/Header/Header.js

```diff
@@ -2,9 +2,9 @@ import "./Header.css";
 
 const Header = () => {
   return (
-    <div>
-      <span className="Header">Movie APP</span>
-    </div>
+    <span onClick={() => window.scroll(0, 0)} className="Header">
```

Comment:
What are you trying to achieve here with `window.scroll`?

Reply:
If you click on the header the function goes back in basic position.